Data/Map/Lazy.hs

```diff
@@ -88,24 +88,37 @@ module Data.Map.Lazy (
     -- * Map type
     Map              -- instance Eq,Show,Read
 
-    -- * Operators
-    , (!), (!?), (\\)
+    -- * Construction
+    , empty
+    , singleton
+    , fromSet
+
+    -- ** From Lists
```

Comment:
Why are these in a separate section here but under construction for sets? One way or the other, I think they should be consistent.

Reply:
I went back and fourth with this a few times. The number of fromList functions for sets is relatively small but maps have A LOT! I would argue that some of these are so specialized (`fromDistinctAscKey` and friends) that they don't belong in the core containers package at all and would be better hosted in containers-extra, but that's a separate discussion :)

I've taken your suggestion below and added sections for the different fromList variants.